src/views/Practice/Practice.jsx

```
@@ -500,9 +489,16 @@ const Practice = () => {
   }, []);
 
   useEffect(() => {
-    localStorage.setItem("mechanism_id", (mechanism && mechanism.id) || "");
+    setLocalData("mechanism_id", (mechanism && mechanism.id) || "");
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)